run_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Run unit tests
+printf "\n--------------------- Unit Tests ----------------------\n"
+py.test inspirehep tests/unit
+unit_tests_exit_code=$?
+
+# Make documentation
+printf "\n\n--------------- Creating Documentation ----------------\n"
+make -C docs html
+create_docs_exit_code=$?
+
+if [ unit_tests_exit_code != 0 ]
+then
+	exit 1
+elif [ create_docs_exit_code != 0 ]
+then
+	exit 2
```

Comment:
If you want to use special exit codes for each case, you should keep in mind that there are 3 different cases, failing the docs, failing the unit, and failing both.